samples/nrf9160/nrf_cloud_mqtt_multi_service/src/connection.c

```diff
@@ -453,7 +453,10 @@ static void update_shadow(void)
 	struct nrf_cloud_svc_info_ui ui_info = {
 		.gnss = location_tracking_enabled(),
 		.temperature = IS_ENABLED(CONFIG_TEMP_TRACKING),
-		.logs = IS_ENABLED(CONFIG_NRF_CLOUD_LOG_BACKEND)
+		.log = IS_ENABLED(CONFIG_NRF_CLOUD_LOG_BACKEND) &&
+		       IS_ENABLED(CONFIG_LOG_BACKEND_NRF_CLOUD_OUTPUT_TEXT),
```

Comment:
where are `CONFIG_LOG_BACKEND_NRF_CLOUD_OUTPUT_TEXT` and `CONFIG_LOG_BACKEND_NRF_CLOUD_OUTPUT_DICTIONARY` defined?

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/logging/Kconfig.template.log_format_config

Reply:
In other words, it's magic.